pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -51,7 +51,6 @@ var (
 	preboundPVC         = makeTestPVC("prebound-pvc", "1G", "", pvcPrebound, "pv-node1a", "1", &waitClass)
 	preboundPVCNode1a   = makeTestPVC("unbound-pvc", "1G", "", pvcPrebound, "pv-node1a", "1", &waitClass)
 	boundPVC            = makeTestPVC("bound-pvc", "1G", "", pvcBound, "pv-bound", "1", &waitClass)
-	boundPVC2           = makeTestPVC("bound-pvc2", "1G", "", pvcBound, "pv-bound2", "1", &waitClass)
```

Comment:
https://github.com/kubernetes/kubernetes/pull/81680 already covers this

Reply:
Got it.

@praseodym Can you revert the change in latest commit on pkg/controller/volume/scheduling (as well as the entry in .staticcheck_failures)

Reply:
Done (and rebased)!